synctl

```diff
@@ -292,11 +318,12 @@ def main():
             write("All processes exited; now restarting...")
 
     if action == "start" or action == "restart":
+        errors = 0
         if start_stop_synapse:
             # Check if synapse is already running
             if os.path.exists(pidfile) and pid_running(int(open(pidfile).read())):
                 abort("synapse.app.homeserver already running")
-            start(configfile, bool(options.daemonize))
+                errors += int(start(configfile, bool(options.daemonize)))
```

Comment:
the cast of a bool to an int is a bit horrible, not least because I think it gives the wrong result.

Make it explicit:

```
success = start(...)
if not success:
    errors += 1
```

Reply:
Right, I flipped the boolean in my head.